pkg/config/config.go

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) 2020 Red Hat, Inc.
+// Copyright Contributors to the Open Cluster Management project
+package config
+
+import (
+	"os"
+	"path/filepath"
+	"strconv"
+
+	"github.com/golang/glog"
+)
+
+const (
+	DEFAULT_SERVICE_PORT = ":3010"
```

Comment:
Use a different port to avoid collisions if we are running the search aggregator locally at the same time.

Reply:
done